src/devices/tuya.ts

```diff
@@ -3740,6 +3740,34 @@ const definitions: Definition[] = [
             await reporting.bind(device.getEndpoint(4), coordinatorEndpoint, ['genOnOff']);
         },
     },
+    {
+        fingerprint: [
+            {modelID: 'TS0004', manufacturerName: '_TZ3000_nivavasg'},
+        ],
+        model: 'TS0004_switch_4_gang_with_backlight',
+        vendor: 'TuYa',
+        description: 'Smart light switch - 4 gang with backlight and neutral wire',
+        extend: tuya.extend.switch({powerOnBehavior2: true, backlightModeOffOn: true, endpoints: ['l1', 'l2', 'l3', 'l4']}),
```

Comment:
Can you add this to the existing `TS0004` instead?

Reply:
Yes I can, I just made that way because I see others PR's creating a new one.

I'll add TS0004.

Reply:
@Koenkk I updated the PR using the existent TS0004.